src/internal/collection/postgres_collection.go

```diff
@@ -72,14 +77,21 @@ func WithNotFoundMessage(format func(interface{}) string) Option {
 	}
 }
 
+func WithListBufferCapOverride(cap int) Option {
```

Comment:
This would probably be fine as `WithListBufferCapacity`, the override is implicit when using the functional option.